torchvision/datapoints/__init__.py

```diff
@@ -22,7 +22,7 @@ def wrap(wrappee, *, like, **kwargs):
 
     Args:
         wrappee (Tensor): The tensor to convert.
-        like (Datapoint): The
+        like (Datapoint): The reference. ``wrappee`` will be converted into the same subclass as ``like``.
```

Comment:
As discussed offline, this needs to replaced with something else to avoid the garbling by sphinx.